### PR TITLE
Clean up About page

### PR DESCRIPTION
**Pull request by coyotebush**
I started by adding a link to the History page that people can understandably never find, then found lots of other things to fix.

### CODE REVIEW

File: about/index.md [hunk 1]
```diff
@@ -3,14 +3,15 @@ title: About
 author: the CPLUG God
 layout: page
 ---
```
Comment (icco) [inline about/index.md:5]: Could you add a sub header or something with prominent links to alunni and history pages? They're hard to find burried in the text.

Comment (coyotebush) [inline about/index.md:5]: Hmm, this page is short enough that I'd expect people to read straight through it. Maybe expand the link text (so `[esteemed alumni]` and `[history of CPLUG]`), and/or do something with the second paragraph which is kind of redundant with the Contact page.

Comment (icco) [inline about/index.md:5]: Most people, when they see only text, assume there aren't any sub pages.

Comment (coyotebush) [inline about/index.md:5]: Fair enough. I'm not sure a heading suggests that either, but we could probably put these in a bulleted list.

**Comment by icco** [comment]
:+1:

**Pull request merged**